### PR TITLE
Rev42/tfpdf

### PR DESCRIPTION
**Pull request by timothygwebb**
closing #633

### CODE REVIEW

**Comment by evrifaessa** [comment]
@timothygwebb, Just did a fresh install from rev42/tfpdf branch. Trying to download an invoice as PDF (by navigating to `/invoice/pdf/xxxxx`) results in a blank white page. No outputs, no downloads, no responses at all. Are we sure that this pull request is going to solve the issue?

**Comment by rndevuk** [comment]
A quick look at this pull request and it should at least bring it back to where it was... but we are still going to need to update the pdf system

**Comment by timothygwebb** [comment]
> @timothygwebb, Just did a fresh install from rev42/tfpdf branch. Trying to download an invoice as PDF (by navigating to `/invoice/pdf/xxxxx`) results in a blank white page. No outputs, no downloads, no responses at all. Are we sure that this pull request is going to solve the issue?
Did you check your logs for any errors it would be greatly appreciated to know if it was a rendering issue or a font definition issue. I need to know source of your issue Apache/Nginx or  tfpdf. It resolved fine for me on Nginx.

**Comment by timothygwebb** [comment]
> A quick look at this pull request and it should at least bring it back to where it was... but we are still going to need to update the pdf system

@rndevuk I am currently working on implementing https://github.com/tecnickcom/tc-lib-pdf

File: src/bb-library/info.htm [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```
Comment (fordnox) [inline src/bb-library/info.htm:1]: what is the purpose of this file?

**Review by fordnox — CHANGES_REQUESTED** [review]
make at least one test where pdf is generated

File: src/bb-library/info.htm (hunk 1 above)
Comment (timothygwebb) [reply src/bb-library/info.htm:1]: @fordnox It is a file from the maintainer that is letting people know more detailed information about it such as it requires ext-mbstring, what license it is using, along with contact information. I do not know why it was left or committed in the maintainers language. Do you want me to reinterpret in english or just remove the file.

**Comment by timothygwebb** [comment]
> make at least one test where pdf is generated

@fordnox Will do.

**Comment by evrifaessa** [comment]
> Did you check your logs for any errors it would be greatly appreciated to know if it was a rendering issue or a font definition issue. I need to know source of your issue Apache/Nginx or  tfpdf. It resolved fine for me on Nginx.
Running on a XAMPP stack, Apache. Can't see any errors in both my Apache and PHP logs.

**Comment by evrifaessa** [comment]
Tried again in another server. This also is an Apache server, and it still only returns a blank page.

**Comment by timothygwebb** [comment]
> Tried again in another server. This also is an Apache server, and it still only returns a blank page.

@evrifaessa Thank you for the update. I am working on this still. @fordnox  Thanks for drafting the pull.

**Pull request closed without merging**